faqs/galaxy/datasets_custom_builds.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Adding Custom Builds to database
+area: datasets
+box_type: tip
+layout: faq
+contributors: [jennaj, Nurzhamalyrys]
+---
+
+
+Some tools and functions require that the 'database' attribute that is assigned. A Custom Reference Genome is set up as a Custom Build prior to use. Examples are the tools Featurecounts, Extract Genomic DNA, certain Picard tools, and the functions under Visualization.
+
+Once Custom Build is created, it is added to the list Database/Build: on the dataset 'Edit Attributes' and 'Upload File' tool forms and is available for 'Visualizations'. These can be assigned or used just like any other reference genome.
+
+1. Start with an existing fasta Custom Reference Genome in your history. It is very important make sure the format is correct.
+2. Go to the top "User" menu and select "Custom Builds".
```

Comment:
I think that for consistency with the other FAQs it would be **User** and **Custom Builds**. Same with the keywords in the following steps